hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -27,7 +27,7 @@
 namespace hardware_interface
 {
 
-typedef std::variant<double> HANDLE_DATATYPE;
+typedef std::variant<bool, double, int8_t, uint8_t> HANDLE_DATATYPE;
```

Comment:
If we already use variants, then this should use vectors of `uint8_t` and not individual values.